Templates/CSharp/EdgeDriverTemplate/ProjectTemplates/CSharp/Test/EdgeDriverTemplate.vstemplate

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
+  <TemplateData>
+    <TemplateID>Microsoft.CSharp.UnitTestProject.EdgeDriverTest</TemplateID>
+    <Name>Web Driver Test for Edge (.NET Framework)</Name>
+    <Description>Create an automatic UI test using Edge Driver.</Description>
+    <Icon Package="{52CBD135-1F97-2580-011F-C7CD052E44DE}" ID="531"/>
+    <ProjectType>CSharp</ProjectType>
+    <SortOrder>100</SortOrder>
+    <CreateNewFolder>true</CreateNewFolder>
+    <DefaultName>EdgeDriverTest</DefaultName>
+    <ProvideDefaultName>true</ProvideDefaultName>
+    <CreateInPlace>false</CreateInPlace>
+    <RequiredFrameworkVersion>4.6</RequiredFrameworkVersion>
```

Comment:
Did we figure out whether we need license? I wonder if that is required for the command line scenario where one creates these projects via CLI (not sure if the command line scenario is supported by default for templates in this repo)...

Reply:
I believe these templates only apply VS, not for CLI. If we want it for CLI, then we should make some other changes in template_feed folder in this repo. Also I am not sure about the license.